OpenRA.Mods.Common/Widgets/Logic/Ingame/Hotkeys/CycleProductionActorsHotkeyLogic.cs

```diff
@@ -25,6 +25,8 @@ public class CycleProductionActorsHotkeyLogic : SingleHotkeyBaseLogic
 		readonly Selection selection;
 		readonly World world;
 
+		private string clickSound = ChromeMetrics.Get<string>("ClickSound");
```

Comment:
Please also expose this to yaml via the logicArgs.  Take a look at how the logic hotkeys are defined for an example to work from.  This applies to the other logic types too.

Reply:
Also note that `private` should not be explicitly declared - it is the default for fields.

Reply:
Need further explanation why this wrong here, but correct in all the other files.

Reply:
The other files are `Widget`s, which the engine automatically expose to yaml overrides.